bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
it's going to need editing one way or another. either in a merge conflict or in a pr.